src/ui/common/src/components/integrations/cards/card.tsx

```diff
@@ -29,67 +23,69 @@ import { SlackCard } from './slackCard';
 import { SnowflakeCard } from './snowflakeCard';
 import { SparkCard } from './sparkCard';
 
-type DataProps = {
-  dataPreviewInfo: DataPreviewInfo;
-};
+// TODO: this component appears to not be used anymore.
 
-export const DataCard: React.FC<DataProps> = ({ dataPreviewInfo }) => {
```

Comment:
Shall we remove? May be a question for the larger team here.

I think we should remove if this is no longer used.